plugins/plugin-template/src/main.php

```diff
@@ -1,10 +1,14 @@
 <?php
 
-chdir(__DIR__);
+// __DIR__ means "directory of _this_ file, i.e. where main.php is"
+define('EXAMPLE_PLUGIN_BASE_DIRECTORY', __DIR__);
+// now we are sure that we're currently running in this directory
+chdir(EXAMPLE_PLUGIN_BASE_DIRECTORY);
```

Comment:
IIRC we've handled this directly in UCRM already, so it's not needed right? @Piskvor